components/usage/pkg/scheduler/reporter.go

```diff
@@ -16,26 +16,26 @@ const (
 )
 
 var (
-	reconcileStartedTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+	jobStartedSeconds = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: namespace,
 		Subsystem: subsystem,
-		Name:      "reconcile_started_total",
-		Help:      "Number of usage reconciliation runs started",
-	}, []string{})
+		Name:      "scheduler_job_started",
```

Comment:
Prometheus [metric naming conventions](https://prometheus.io/docs/practices/naming/)  suggest this might be better to have the unit as a suffix
```suggestion
		Name:      "scheduler_jobs_started_total",
```

Reply:
thanks, good spot